index.js

```diff
@@ -20,14 +20,6 @@ module.exports = function (a, b) {
 	var aLen = a.length;
 	var bLen = b.length;
 
-	if (aLen === 0) {
-		return bLen;
-	}
-
-	if (bLen === 0) {
```

Comment:
So what you mean is, that if `bLen` is `0`, then `aLen` must be `0` as well and the check at line `56` will kick in again and returns `0`?